pages/searching/parameters.rst

```diff
@@ -0,0 +1,38 @@
+Parameters
```

Comment:
add before:

```
.. _search_parameters:

```

That we have the ability to use references to this page by that.